### DISCUSSION TITLE
Unable to export custom values

### DISCUSSION THREAD

**Discussion by leeadh** [apache/airflow#21512] [Ideas]
### Description

Im trying to export custom values

    export_custom_task = BashOperator(
        task_id="bash_task",
        bash_command="exporting values",
        env={
            'CUSTOM_ENV': 'CUSTOM_VALUE',
            **os.environ
        },
        dag=dag
    )

    get_env_task = BashOperator(
        task_id="sample_task",
        bash_command="echo $CUSTOM_ENV",
        dag=dag
    )

but when i try to echo the variable, nothing returns. Perhaps it will be good to build in the docs how to do this as I need exported values for a task. 

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by mik-laj** [comment]
Each operators is executed as separate process, so you need to explicitly exchange data using xcom. http://airflow.apache.org/docs/apache-airflow/stable/concepts/xcoms.html
